code/__DEFINES/research.dm

```diff
@@ -77,11 +78,19 @@
 ///R&D point value for a maxcap bomb. Can be adjusted if need be. Current Value Cap Radius: 100
 #define TECHWEB_BOMB_POINTCAP 50000
 
-///Research point values for slime extracts, see: [/modules/research/xenobiology/xenobio_camera][xenobio_camera]
-#define SLIME_RESEARCH_TIER_0 100
-#define SLIME_RESEARCH_TIER_1 500
-#define SLIME_RESEARCH_TIER_2 1000
-#define SLIME_RESEARCH_TIER_3 1500
-#define SLIME_RESEARCH_TIER_4 2000
-#define SLIME_RESEARCH_TIER_5 2500
-#define SLIME_RESEARCH_TIER_RAINBOW 5000
+///Research point values for slime extracts, see the R&D console's attackby code
+#define SLIME_RESEARCH_TIER_0 500
+#define SLIME_RESEARCH_TIER_1 1500
+#define SLIME_RESEARCH_TIER_2 3000
+#define SLIME_RESEARCH_TIER_3 6000
+#define SLIME_RESEARCH_TIER_4 9000
+#define SLIME_RESEARCH_TIER_5 15000
+#define SLIME_RESEARCH_TIER_RAINBOW 20000
+
+///Research point values for plant discoveries
+#define PLANT_RESEARCH_TIER_0 500
+#define PLANT_RESEARCH_TIER_1 1500
+#define PLANT_RESEARCH_TIER_2 3000
+#define PLANT_RESEARCH_TIER_3 6000
+#define PLANT_RESEARCH_TIER_4 9000
+#define PLANT_RESEARCH_TIER_5 15000
```

Comment:
I'm not sure how plant discoveries work, if they are only once per techweb I'm fine with this.

Reply:
both plant discoveries and slime extracts are "discoveries" that can only be redeemed once. That being said, I do need help connecting their "discovered items" lists to ships directly, so that you can't do the old reconstruct the console trick to just redeem everything again